### PR TITLE
Fix logging

### PR DESCRIPTION
**Pull request by tcanabrava**

### CODE REVIEW

File: src/atcore.h [hunk 1]
```diff
@@ -4,10 +4,15 @@
 #include <QObject>
 #include <QList>
 #include <QSerialPortInfo>
+#include <QLoggingCategory>
 
 class ProtocolLayer;
 struct AtCorePrivate;
 
+Q_DECLARE_LOGGING_CATEGORY(Received)
+Q_DECLARE_LOGGING_CATEGORY(Sended)
```
Comment (sithlord48) [inline src/atcore.h:13]: please use "Sent" instead of sended

File: src/atcore.cpp [hunk 2]
```diff
@@ -10,6 +10,10 @@
 #include <QTime>
 #include <QTimer>
 
+Q_LOGGING_CATEGORY(Received, "Received")
+Q_LOGGING_CATEGORY(Sended, "Sended")
```
Comment (sithlord48) [inline src/atcore.cpp:14]: Please use "Sent" instead of Sended

File: src/atcore.cpp [hunk 3]
```diff
@@ -48,18 +51,20 @@ void AtCore::findFirmware(const QByteArray& message)
 {
     static int initialized = 0;
     if (!initialized) {
-        QTimer::singleShot(500, this, [=]{qDebug() << "Sending M115"; d->protocol->pushCommand("M115");});
+        QTimer::singleShot(500, this, [=]{
+            qCDebug(FirmwarePlugin) << "Sending M115";
```
Comment (sithlord48) [inline src/atcore.cpp:55]: Do we want strings like this to be translated?

**Review by sithlord48 — APPROVED** [review]

**Comment by patrickelectric** [comment]
@tcanabrava please close this PR.

**Pull request closed without merging**